src/Uno.Extensions.Hosting.UI/UnoHost.cs

```diff
@@ -22,19 +22,22 @@ internal static IHostBuilder CreateDefaultBuilder(Assembly applicationAssembly,
 				catch
 				{
 					// This will throw an exception on WinUI if unpackaged, so dataFolder will be null
+					// Can also be null on Linux FrameBuffer
 				}
-#if WINUI && WINDOWS
-				var appName = Assembly.GetEntryAssembly()?.GetName().Name ?? string.Empty;
+
 				if (string.IsNullOrWhiteSpace(dataFolder))
 				{
+					var appName = Assembly.GetEntryAssembly()?.GetName().Name ?? string.Empty;
```

Comment:
Should we maybe have a default value here so that we don't store things in the Local Application Data directory directly? i.e. "unoapp"?